<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>42</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(242, 198, 198);
border-radius:20px;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color:rgb(0, 0, 0)</string>
        </property>
        <property name="text">
         <string>Sales report for the last month </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="minimumSize">
        <size>
         <width>174</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_2{
background-color: #D9D9D9;
border-radius: 20px; 
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color:#000000</string>
          </property>
          <property name="text">
           <string>From memberships :</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_7" native="true">
       <property name="minimumSize">
        <size>
         <width>174</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_7{
background-color: #D9D9D9;
border-radius: 20px; 
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">color:#000000</string>
          </property>
          <property name="text">
           <string>$XXXX.XX</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QWidget" name="widget_6" native="true">
       <property name="minimumSize">
        <size>
         <width>174</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_6{
background-color: #D9D9D9;
border-radius: 20px; 
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">color:#000000</string>
          </property>
          <property name="text">
           <string>From Supplies :</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_8" native="true">
       <property name="minimumSize">
        <size>
         <width>174</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_8{
background-color: #D9D9D9;
border-radius: 20px; 
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">color:#000000</string>
          </property>
          <property name="text">
           <string>$XXXX.XX</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="minimumSize">
        <size>
         <width>174</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_3{
background-color: #D9D9D9;
border-radius: 20px; 
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">color:#000000</string>
          </property>
          <property name="text">
           <string>From workout sessions:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_9" native="true">
       <property name="minimumSize">
        <size>
         <width>174</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_9{
background-color: #D9D9D9;
border-radius: 20px; 
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">color:#000000</string>
          </property>
          <property name="text">
           <string>$XXXX.XX</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="minimumSize">
        <size>
         <width>174</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_4{
background-color: #D9D9D9;
border-radius: 20px; 
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color:#000000</string>
          </property>
          <property name="text">
           <string>From Tool res :</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_10" native="true">
       <property name="minimumSize">
        <size>
         <width>174</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_10{
background-color: #D9D9D9;
border-radius: 20px; 
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_10">
          <property name="styleSheet">
           <string notr="true">color:#000000</string>
          </property>
          <property name="text">
           <string>$XXXX.XX</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="minimumSize">
        <size>
         <width>174</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_5{
background-color: #EA7171;
border-radius: 20px; 
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">color:#000000</string>
          </property>
          <property name="text">
           <string>Expenses:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_11" native="true">
       <property name="minimumSize">
        <size>
         <width>174</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_11{
background-color: #EA7171;
border-radius: 20px; 
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">color:#000000</string>
          </property>
          <property name="text">
           <string>$XXXX.XX</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QWidget" name="widget_12" native="true">
       <property name="minimumSize">
        <size>
         <width>174</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_12{
background-color: #5AB62C;
border-radius: 20px; 
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_12">
          <property name="styleSheet">
           <string notr="true">color:#000000</string>
          </property>
          <property name="text">
           <string>Total profit:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_13" native="true">
       <property name="minimumSize">
        <size>
         <width>174</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_13{
background-color: #5AB62C;
border-radius: 20px; 
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_13">
          <property name="styleSheet">
           <string notr="true">color:#000000</string>
          </property>
          <property name="text">
           <string>$XXXX.XX</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_20">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FC4A4A;    /* bright red */
    color: white;
    border: none;                 /* remove default border */
    border-radius: 20px;          /* smooth rounded corners */
    font-weight: bold;
    font-size: 16px;
    padding: 10px 20px;
    transition: all 0.2s ease;   /* smooth hover/press transition */
}

/* Hover effect: slightly darker red */
QPushButton:hover {
    background-color: #E03B3B;
}

/* Pressed effect: even darker red */
QPushButton:pressed {
    background-color: #C13030;
    padding-left: 18px;   /* subtle &quot;press&quot; effect */
    padding-top: 12px;
}
</string>
       </property>
       <property name="text">
        <string>Go Back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
